pkg/proxy/proxy.go

```diff
@@ -59,15 +59,15 @@ func (p *proxy) Setup(ctx context.Context, obj *unstructured.Unstructured) error
 func (p *proxy) setupDaemonSet(ctx context.Context, obj *unstructured.Unstructured) error {
 	containers, found, err := unstructured.NestedSlice(obj.Object, "spec", "template", "spec", "containers")
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to find slice spec/template/spec/containers in daemonset %s/%s: %w", obj.GetNamespace(), obj.GetName(), err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to find slice spec.template.spec.containers in daemonset %s/%s: %w", obj.GetNamespace(), obj.GetName(), err)
```

Don't you prefer it? we could confuse it with `namespace/name` and also, this is how I did it in some pakcages.

Reply:
fixed